build.gradle

```diff
@@ -47,7 +47,9 @@ dependencies {
     implementation files('lib/ojdbc8.jar')
 
     /* *** OVERRIDDEN VULNERABLE TRANSITIVE DEPENDENCIES *** */
-    implementation 'org.yaml:snakeyaml:2.1' //spring-boot-starter
+    implementation 'org.yaml:snakeyaml:2.1'
+    testImplementation 'org.springframework.cloud:spring-cloud-stream-test-binder'
```

Comment:
Is this library utilized?

Reply:
It is not. I'll remove and update the branch.